.gitmodules

```diff
@@ -25,3 +19,33 @@
 [submodule "roles/jekyll"]
 	path = roles/jekyll
 	url = https://github.com/void-ansible-roles/jekyll.git
+[submodule "roles/cinnamon"]
+	path = roles/cinnamon
+	url = https://github.com/void-ansible-roles/cinnamon.git
+[submodule "roles/lxde"]
+	path = roles/lxde
+	url = https://github.com/void-ansible-roles/lxde.git
+[submodule "roles/xorg"]
+	path = roles/xorg
+	url = https://github.com/void-ansible-roles/xorg.git
+[submodule "roles/lightdm"]
+	path = roles/lightdm
+	url = https://github.com/void-ansible-roles/lightdm.git
+[submodule "roles/lightdm-webkit-greeter"]
+	path = roles/lightdm-webkit-greeter
+	url = https://github.com/void-ansible-roles/lightdm-webkit-greeter.git
+[submodule "roles/i3"]
+	path = roles/i3
+	url = https://github.com/void-ansible-roles/i3.git
+[submodule "roles/awesome"]
+	path = roles/awesome
+	url = https://github.com/void-ansible-roles/awesome.git
+[submodule "roles/xfce4"]
+	path = roles/xfce4
+	url = https://github.com/void-ansible-roles/xfce4.git
+[submodule "roles/bspwm"]
+	path = roles/bspwm
+	url = https://github.com/void-ansible-roles/bspwm.git
+[submodule "roles/dbus"]
+	path = roles/dbus
+	url = https://github.com/void-ansible-roles/dbus.git
```

Comment:
Fair enough.  That said, a .gitconfig is also a machine-readable file that can be modified solely with the cli (`git config --global option.name option.value`) and many people still alphabetize that.  We've certainly had to manually edit the .gitmodules file manually before.